app/src/main/java/org/sopt/soptandroidseminar/view/SignInActivity.kt

```diff
@@ -0,0 +1,64 @@
+package org.sopt.soptandroidseminar.view
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.activity.result.contract.ActivityResultContracts
+import org.sopt.soptandroidseminar.R
+import org.sopt.soptandroidseminar.databinding.ActivitySignInBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignInBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignInBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        goToSignUpActivity()
+        loginCheckEvent()
+    }
+
+    private fun goToSignUpActivity() {
+        binding.btnRegister.setOnClickListener {
+            val intent = Intent(this, SignUpActivity::class.java)
+            signUpActivityLauncher.launch(intent)
+        }
+    }
+
+    private fun loginCheckEvent() {
+        binding.btnLogin.setOnClickListener {
+            if (checkInputText()) {
+                showToast("로그인 실패")
+            } else {
+                goToHomeActivity()
+            }
+        }
+    }
+
+    private val signUpActivityLauncher = registerForActivityResult(
+        ActivityResultContracts.StartActivityForResult()
+    ) {
+        if (it.resultCode == Activity.RESULT_OK) {
+            val id = it.data?.getStringExtra("userId")
+            val pw = it.data?.getStringExtra("userPw")
+
+            binding.editId.setText(id)
+            binding.editPw.setText(pw)
+
+        }
+    }
+
+    private fun checkInputText(): Boolean {
+        return binding.editId.text.isNullOrBlank() || binding.editPw.text.isNullOrBlank()
+    }
```

Comment:
```suggestion
    private fun checkInputText(): Boolean = binding.editId.text.isNullOrBlank() || binding.editPw.text.isNullOrBlank()
```
이렇게만 작성하셔도 괜찮을 것 같습니다~!!